package.json

```diff
@@ -26,14 +26,12 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.6.0",
-    "@aurodesignsystem/auro-dropdown": "^2.9.1",
-    "@aurodesignsystem/auro-input": "^2.10.2",
     "@aurodesignsystem/webcorestylesheets": "^4.0.0"
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.12.1",
-    "@aurodesignsystem/auro-dropdown": "^2.9.4",
-    "@aurodesignsystem/auro-input": "^2.10.4",
+    "@aurodesignsystem/auro-dropdown": "^2.9.7",
+    "@aurodesignsystem/auro-input": "^2.13.2",
```

Comment:
These need to be direct dependencies. As dev dependencies, they will not be included in the bundle and not be available to be used for the extension.